### ISSUE TITLE
Usernames cannot have dots in them

### ISSUE THREAD

**Issue by njt1982** [harshq/react-native-mentions#14]
https://github.com/harshq/react-native-mentions/blob/b1aac08c91e94b5cac3ddfd3df908da0aa71d828/src/MentionsTextInput.js#L77

This assumes all usernames match that syntax.

We have a username with `@first.lastname`. The result of this code is that the keyword passed to our triggerCallback does not contain anything including and after the dot.

Suggest that the regex is exposed as a param and defaults to its current value?

**Comment by harshq** [comment]
Good idea, You are welcome to send a PR if you have fixed it already. I'll fix this with the next release otherwise :)

**Comment by mrazadar** [comment]
Hey just published a new [react-native-mentions](https://www.npmjs.com/package/react-native-mentions-editor) package. Check it out.
